app/controllers/manageBusinesses/add/AddBusinessNameController.scala

```diff
@@ -74,25 +74,15 @@ class AddBusinessNameController @Inject()(val authorisedFunctions: AuthorisedFun
     }
   }
 
-  def show(): Action[AnyContent] = auth.authenticatedAction(isAgent = false) {
+  def show(isAgent: Boolean, isChange: Boolean): Action[AnyContent] = auth.authenticatedAction(isAgent) {
     implicit user =>
       handleRequest(
-        isAgent = false,
+        isAgent,
```

Comment:
please use named arguments,
e.g:
isAgent = isAgent,
backUrl = getBackUrl(isAgent = false, isChange = false),
isChange = isChange

it makes it easier to differentiate arguments of the same type (Booleans in this case)

Reply:
Done